widgets/common/src/deployModal/GenericDeployModal.tsx

```diff
@@ -132,6 +133,11 @@ type GenericDeployModalProps = {
      * Deployment Name input help description
      */
     deploymentNameHelp?: string;
+
+    /**
+     * Filter listing blueprints rules
```

Comment:
Probably more accurate would be:
```suggestion
     * Filter rules for blueprints listing
```
WDYT?

Reply:
[4afcb82](https://github.com/cloudify-cosmo/cloudify-stage/pull/1935/commits/4afcb82bf37f81a9833b88ea1e5000f376a02831)